extra/3-magic-8-ball.js

```diff
@@ -58,10 +110,28 @@ function shakeBall() {
 
   This function should expect to be called with any value which was returned by the shakeBall function.
 */
-function checkAnswer(answer) {
+function checkAnswer(string) {
   //Write your code in here
+  for (let i=0; i<4;i++){
+    let list=listsOfAnswers[i];
+     for (let j = 0; j < 5; j++) {
```

Comment:
Good attempt at variable naming here! A little wordy perhaps, but easier for me to work out what's going on in your code very quickly.